dynamic-config/config-kie/src/main/java/org/apache/servicecomb/config/kie/client/KieClient.java

```diff
@@ -84,6 +101,8 @@ private void deployConfigClient() throws InterruptedException {
     Vertx vertx = VertxUtils.getOrCreateVertxByName("kie", vertxOptions);
 
     HttpClientOptions httpClientOptions = new HttpClientOptions();
+    httpClientOptions.setKeepAlive(true);
+    httpClientOptions.setIdleTimeout(SOCKET_TIMOUT);
```

Comment:
Please look at RestTransportClient to setting HttpClientOptions, if keepAliveTimeout not set, IdelTimeout may not work as expected. SOCKET_TIMEOUT should better be IdleTimeout and maybe better add a configuration for this.

Reply:
keepAliveTimeout default is 60s

Reply:
I see the vert.x said :

>    * Set the keep alive timeout for HTTP/1.x, in seconds.
>    * This value determines how long a connection remains unused in the pool before being evicted and closed. 

I think the default value is ok